error.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Server Error</title>
+    <link rel="stylesheet" href="/error/static/styles.css">
+    <link rel="stylesheet" href="/error/static/font_raleway.css">
+</head>
+
+<body>
+    <main>
+        <h1>
+            This site is currently unavailable due to a server error.
+            Please try again in a few minutes.
+        </h1>
+    </main>
+</body>
+
+</html>
```

Comment:
Style nit:  Having no newline at the enf of the file results in spurious diffs, since some editors automatically append one when saving.  (This is completely optional to address.)